tox.ini

```diff
@@ -2,10 +2,7 @@
 min_version = 4.0
 
 envlist =
-    py{38,39,310}-django{32,41}-wagtail{41,50,51}
-    py{311}-django{41,42}-wagtail{50,51}
-    py{311,312}-django{41,42}-wagtail{50,51,52}
-    py{311,312}-django{42,50}-wagtail60
+    py{38,39,310,311,312}-django{42,50}-wagtail{52,60}
```

Comment:
Can you add Python 3.12, Wagtail 6, and remove Wagtail 4 from PyPI classifiers https://github.com/torchbox/wagtail-grapple/blob/main/pyproject.toml#L17 ?